### DISCUSSION TITLE
Unity Project Example?

### DISCUSSION THREAD

**Discussion by antiero** [Marfusios/websocket-client#131] [Q&A]
Hello, has anyone used this code in a Unity Project, and if so, are they able to share an example please/

**Comment by Marfusios** [comment]
Hey @antiero , 

I have used this library in Unity for one of my clients. 
Unfortunately, I don't have the source code anymore. 

If I remember correctly, I had to add dll files manually, and then I had to implement a simple queue structure for received messages. 
Later in the game loop, in `Update()` method I processed the messages from the queue.
